### PR TITLE
Addd 2:1 resolution

### PR DESCRIPTION
**Pull request by rohanhc**

### CODE REVIEW

**Comment by rampr** [comment]
we should solve the splash screen issue also for these devices along with this

**Comment by rohanhc** [comment]
Done

File: build.js [hunk 1]
```diff
@@ -680,6 +680,7 @@ var SPLASH_FILES = [
   'portrait1024',
   'portrait1136',
   'portrait2048',
+  'portrait2960',
```
Comment (rampr) [inline build.js:683]: what happens if the game doesn't specify this resolution ?

Comment (rohanhc) [reply build.js:683]: We also specify in the manifest @rampr I won't add if manifest does not have the resolution

**Review by bijoshtj — APPROVED** [review]

File: build.js (hunk 1 above)
Comment (rampr) [reply build.js:683]: okies

**Review by rampr — APPROVED** [review]

**Review by vivekb91 — APPROVED** [review]

**Pull request merged**